app/controllers/sign_up/completions_controller.rb

```diff
@@ -0,0 +1,24 @@
+module SignUp
+  class CompletionsController < ApplicationController
```

Comment:
sure we can...which preconditions should be required? that someone is logged in?